app/models/ingest_job.rb

```diff
@@ -535,7 +535,7 @@ def generate_error_email_body(email_type: :user)
       end
     end
 
-    if error_contents.empty? || email_type == :dev
+    if !error_contents.present? || email_type == :dev
```

Comment:
Naive questions, after [reading up](https://gist.github.com/pythonicrubyist/8114720) a bit:

1. Is this refinement intended to handle when `error_contents` is `nil`?
2. As a rule of thumb, is `!error_contents.present?` preferred over `foo.empty?`?

Reply:
yes, this is to fix https://sentry.io/organizations/broad-institute/issues/1895999172/?project=1424198&query=is%3Aunresolved&statsPeriod=14d

Reply:
Got it -- helpful, thanks!